tutorials/debugging.md

```diff
@@ -0,0 +1,14 @@
+# Debugging CRI-O
```

Comment:
Let's link this document from the README.md

Reply:
updated!